portfolio/src/main/webapp/script.js

```diff
@@ -61,6 +61,13 @@ function createCommentElement(comment) {
   const textElement = document.createElement('span');
   textElement.innerText = comment.text;
 
+  // i think this is creating an email element? Not sure what "span" is...
+  const emailElement = document.createElement('span');
```

Comment:
This will print the email right next to the comments.  Span is a box like Div, but doesn't break lines like Div does.  ie. you can have multiple Spans on the same line.  More info:  https://www.w3schools.com/tags/tag_span.asp